src/pages/Profile/index.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { compose, pathOr, omit, mergeRight } from 'ramda';
+import { message, Form } from 'antd';
+
+import PerfilContainer from '../../containers/Profile';
+import { updateSenha } from '../../services/password';
+
+const PerfilDash = ({ username, typeAccount }) => {
+  const [editar, setEditar] = useState(false);
+  const [form] = Form.useForm();
+
+  const error = () => {
+    message.error('Os dados do usuário não foram atualizados');
+  };
```

Comment:
esta função só possui uma linha então não precisa das chaves
```suggestion
  const error = () => message.error('Os dados do usuário não foram atualizados');
```